.env.default

```diff
@@ -25,3 +25,4 @@ GITHUB_SITE_TITLE=GitHub
 DEVISE_MODULES='[database_authenticatable,recoverable,rememberable,trackable,validatable,omniauthable]'
 GOOGLE_AUTHENTICATION=true
 GOOGLE_SITE_TITLE=Google
+ERRBIT_NOTICE_DEPRECATION_DAYS=7
```

Comment:
Let's leave this turned off for now by not including it in .env.default. I'd also like to change the word from deprecated to something a bit more scary sounding because this does actually cause `problems` to be destroyed (in a good way). Maybe `ERRBIT_PROBLEM_DESTROY_AFTER_DAYS`